__std__/cells/plutus/packages/buildable-plutus-fragments.nix

```diff
@@ -0,0 +1,21 @@
+# Like the names suggests, this special derivation builds all available derivations
+# inside the plutus cell, and is used by the check-the-flake script.
+
+{ inputs, cell }:
+
+let
+  inherit (cell.library.pkgs) lib releaseTools;
+
+  fragments =
+    cell.devshells //
```

Comment:
Yes I was very mistaken about build times.
I say we just keep the `pre-commit` check as a GitHub action, and leave everything else in Cicero?

Reply:
Hydra gives bad errors if there's an evaluation failure, which is why we had this in the first place. Are the errors we'll get from Cicero better? I think probably, since we'll just see the eval error directly. So it may indeed just be fine.